### ISSUE TITLE
NameError occured while adding invalid url

### ISSUE THREAD

**Issue by ganeshhubale** [pythonpune/readit#129]
```
[ganeshhubale@localhost ~]$ readit -t bluejeans www.bluejeans.com/
('************', '\nInvalid URL\n', '***********')
Still you want to update: Yes/No --> yes
Traceback (most recent call last):
  File "/home/ghubale/.local/bin/readit", line 11, in <module>
    load_entry_point('readit==0.2', 'console_scripts', 'readit')()
  File "/home/ghubale/.local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/ghubale/.local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/ghubale/.local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ghubale/.local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/ghubale/.local/lib/python2.7/site-packages/readit/cli.py", line 105, in main
    tag_option_yes_no(tag_name, tagged_url)
  File "/home/ghubale/.local/lib/python2.7/site-packages/readit/cli.py", line 143, in tag_option_yes_no
    option = input("Still you want to update: Yes/No --> ")
  File "<string>", line 1, in <module>
NameError: name 'yes' is not defined
```

**Comment by vishalbidwe** [comment]
(venv) [vishal@localhost readit]$ readit -t bluejeans www.bluejeans.com/
************ 
Invalid URL
 ***********
Still you want to update: Yes/No ? yes
Bookmarked.
(venv) [vishal@localhost readit]$ 
**It is not showing this bug on my machine and stored in db**
readit -v
+====+=====================================+===========+============+==========+
| ID |                 URL                 |    TAG    |    DATE    |   TIME   |
+====+=====================================+===========+============+==========+
| 1  |           www.google.com            |   None    | 2018-11-06 | 11:56:26 |
+----+-------------------------------------+-----------+------------+----------+
| 2  |                linux                | linux.org | 2018-11-06 | 11:59:28 |
+----+-------------------------------------+-----------+------------+----------+
| 3  |       https://www.google.com        |   None    | 2018-11-06 | 12:01:05 |
+----+-------------------------------------+-----------+------------+----------+
| 4  |          www.bluejeans.com          | bluejeans | 2018-11-06 | 14:42:32 |
+----+-------------------------------------+-----------+------------+----------+
| 6  |         www.bluejeans.com/          | bluejeans | 2018-11-06 | 14:45:06 |
+----+-------------------------------------+-----------+------------+----------+

**Comment by ganeshhubale** [comment]
Yes. This issue is resolved by PR #124. I forgot to close this issue.
Thank you.
